lib/rules/prefer-regex-literals.js

```diff
@@ -146,6 +146,7 @@ module.exports = {
         messages: {
             unexpectedRegExp: "Use a regular expression literal instead of the 'RegExp' constructor.",
             replaceWithLiteral: "Replace with an equivalent regular expression literal.",
+            replaceWithLiteralAndFlags: "Replace with an equivalent regular expression literal with flags({{ flags }}).",
```

Comment:
```suggestion
            replaceWithLiteralAndFlags: "Replace with an equivalent regular expression literal with flags '{{ flags }}'.",
```

I think this format would look better.

Also, "equivalent" might be misleading for suggestions where the flags are merged because in that case the suggested regex literal is not equivalent to the original code?

Reply:
Thanks! @mdjermanovic. Maybe seperatesmessages and removing "equivalent" seems good. I'll fix it